build-elements/ff4j-features.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs-extra');
+const concat = require('concat');
+
+(async function build() {
+  const files = [
+    './dist/ff4j-features/runtime-es5.js',
+    './dist/ff4j-features/polyfills-es5.js',
+    './dist/ff4j-features/main-es5.js'
```

Comment:
I think we need to use 2015 as it is a step forward

https://johnpapa.net/es5-es2015-typescript/